.github/workflows/run_cell-type-nonETP-ALL-03.yml

```diff
@@ -91,3 +92,7 @@ jobs:
           Rscript scripts/02-03_annotation.R
           Rscript scripts/04_multipanel_plot.R
           Rscript scripts/05_cluster_evaluation.R
+          Rscript scripts/06_sctype_exploration.R
+          Rscript scripts/07_run_copykat.R
+          Rscript scripts/markerGenes_submission.R
+          Rscript scripts/writeout_submission.R
```

Comment:
Hi @UTSouthwesternDSSR,

I was looking at your PR trying to figure out the previous CI failure. It may be due to the small number of cells in the test data, but it may also be a stochastic failure, as I was able to run the whole workflow on a separate machine with the same test data. Nonetheless, it might be helpful for future debugging to add a few info messages like the ones below to help figure out where we are in the CI process. 

Since you just added a small change, I will wait to see if that passes before proceeding too far!

```suggestion
          printf "\n\nRunning 02-03_annotation.R\n"
          Rscript scripts/02-03_annotation.R
          printf "\n\nRunning 04_multipanel_plot.R\n"
          Rscript scripts/04_multipanel_plot.R
          printf "\n\nRunning 05_cluster_evaluation.R\n"
          Rscript scripts/05_cluster_evaluation.R
          printf "\n\nRunning 06_sctype_exploration.R\n"
          Rscript scripts/06_sctype_exploration.R
          printf "\n\nRunning 07_run_copykat.R\n"
          Rscript scripts/07_run_copykat.R
          printf "\n\nRunning markerGenes_submission.R\n"
          Rscript scripts/markerGenes_submission.R
          printf "\n\nRunning writeout_submission.R\n"
          Rscript scripts/writeout_submission.R
```

Reply:
Sure, thank you! I think it is good to add some comments too. I am still doing some minor change on the script and output.